app/models/claim/base_claim.rb

```diff
@@ -157,8 +157,8 @@ def assessment
 
     scope :dashboard_displayable_states, -> { where(state: Claims::StateMachine.dashboard_displayable_states) }
 
-    scope :total_greater_than_or_equal_to, ->(value) { where { total >= value } }
-    scope :total_lower_than, ->(value) { where { total < value } }
+    scope :total_greater_than_or_equal_to, ->(value) { where(arel_table[:total].gteq(value)) }
```

Comment:
[Not a blocker]  Why switch to Arel syntax?  Was there a gem issue?

Reply:
agree with colin, prefer to avoid arel unless necessary